docs/ReleaseNotesWX.md

```diff
@@ -7,10 +7,26 @@ https://pwsafe.org/. Details about changes to older releases may be found in the
 
 In the following, SFxxxx refers to Bug Reports, Feature Requests and Service Requests in PasswordSafe SourceForge Project tickets, and GHxxxx refers to issues in the PasswordSafe GitHub project.
 
+
+PasswordSafe 1.18 Release XXX 2023
+====================================
+
+Bugs fixed in 1.18
```

Comment:
Nit: probably no need to mention the version here, since it's already mentioned in the release header. Same for "Changes".

Reply:
A fair point, but I'm just matching the style in ReleaseNotes.md.